diff --git a/node_modules/nextra-theme-docs/dist/layout.js b/node_modules/nextra-theme-docs/dist/layout.js
index 57a511f..5e15d53 100644
--- a/node_modules/nextra-theme-docs/dist/layout.js
+++ b/node_modules/nextra-theme-docs/dist/layout.js
@@ -1,4 +1,4 @@
-import { jsx, jsxs } from "react/jsx-runtime";
+import { Fragment, jsx, jsxs } from "react/jsx-runtime";
 import { c as _c } from "react-compiler-runtime";
 import { ThemeProvider } from "next-themes";
 import { SkipNavLink } from "nextra/components";
@@ -7,7 +7,7 @@ import { MobileNav } from "./components/sidebar";
 import { LayoutPropsSchema } from "./schemas";
 import { ConfigProvider, ThemeConfigProvider } from "./stores";
 const Layout = (t0) => {
-  const $ = _c(26);
+  const $ = _c(27);
   let children;
   let themeConfig;
   if ($[0] !== t0) {
@@ -27,7 +27,7 @@ const Layout = (t0) => {
     const {
       data: t12,
       error
-    } = LayoutPropsSchema.safeParse(themeConfig);
+    } = LayoutPropsSchema.safeParse({ ...themeConfig, children });
     data = t12;
     if (error) {
       throw z.prettifyError(error);
@@ -96,8 +96,12 @@ const Layout = (t0) => {
     t3 = $[18];
   }
   let t4;
-  if ($[19] !== banner || $[20] !== nextThemes || $[21] !== t3) {
-    t4 = /* @__PURE__ */ jsxs(ThemeProvider, { ...nextThemes, children: [
+  if ($[19] !== banner || $[20] !== nextThemes || $[21] !== t3 || $[22] !== data?.darkMode) {
+    t4 = data?.darkMode === false ? /* @__PURE__ */ jsxs(Fragment, { children: [
+      t1,
+      banner,
+      t3
+    ] }) : /* @__PURE__ */ jsxs(ThemeProvider, { ...nextThemes, children: [
       t1,
       banner,
       t3
@@ -105,18 +109,19 @@ const Layout = (t0) => {
     $[19] = banner;
     $[20] = nextThemes;
     $[21] = t3;
-    $[22] = t4;
+    $[22] = data?.darkMode;
+    $[23] = t4;
   } else {
-    t4 = $[22];
+    t4 = $[23];
   }
   let t5;
-  if ($[23] !== rest || $[24] !== t4) {
+  if ($[24] !== rest || $[25] !== t4) {
     t5 = /* @__PURE__ */ jsx(ThemeConfigProvider, { value: rest, children: t4 });
-    $[23] = rest;
-    $[24] = t4;
-    $[25] = t5;
+    $[24] = rest;
+    $[25] = t4;
+    $[26] = t5;
   } else {
-    t5 = $[25];
+    t5 = $[26];
   }
   return t5;
 };
